src/app/sprints/sprints.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { SprintDto } from '../declarations/models/sprint-dto';
+import { StoreService } from '../p-common/store.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SprintsService {
+  projectId: number;
+
+  constructor(private http: HttpClient,
+              private storeService: StoreService) {
+    this.projectId = this.storeService.getCurrentProjectId();
+  }
+
+  getSprints(): Observable<SprintDto[]> {
```

Comment:
chyba lepiej, żeby sprintId wyciągnąć do parametru funkcji i nie dodawać storea do serwisu

Reply:
jak już to `projectId`. ale rozumiem o co chodzi.